testData/solutions/win32.json5

```diff
@@ -2075,6 +2075,287 @@
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "ToggleKeysOn": {
+                        "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/common/toggleKeys",
+                            "outputPath": "value"
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": "pvParam.dwFlags.TKF_TOGGLEKEYSON"
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/common/toggleKeys": "ToggleKeysOn.value"
+                }
+            }
+        },
+        "isInstalled": [
+            {
+                "type": "gpii.deviceReporter.alwaysInstalled"
+            }
+        ]
+    },
+
+    "com.microsoft.windows.underlineMenuShortcuts": {
+        "name": "Underline menu shorcuts",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETKEYBOARDCUES",
+                    "setAction": "SPI_SETKEYBOARDCUES",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT",
+                    },
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "UnderlineMenuShorcutsOn": {
+                        "schema": {
+                            "title": "Enable/Disable underlaying menu shortcuts",
+                            "description": "Displays a underline showing which is the shortcut to active a menu item",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "KeyboardPreferenceOn": {
+                        "schema": {
+                            "title": "Keyboard as preferred input method",
+                            "description": "Set the keyboard as the preferred input method",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "UnderlineMenuShorcutsOn": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
```

Comment:
I think `gpii.transformer.booleanToNumber` or even just `fluid.transforms.value`  will be enough.

(same goes for the other uses below)